.devcontainer/Dockerfile

```
@@ -2,5 +2,13 @@ FROM ghcr.io/containerbase/node:18.16.0@sha256:fc48ac97eb5d0dee207d4a336a52d7d3b
 
 USER root
 
+RUN apt update; \
+    apt install -y software-properties-common make g++; \
+    add-apt-repository ppa:deadsnakes/ppa; \
+    apt update; \
+    apt install python3.12; \
+    ln -s /usr/bin/python3 /usr/bin/python
+
+
 # renovate: datasource=npm
-RUN install-tool yarn 1.22.19
+RUN install-tool yarn 1.22.19; yarn install
```

```suggestion
RUN install-tool yarn 1.22.19
RUN yarn install
```
otherwise renovate can't update yarn version.